File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
        result = []
        columnNumber -= 1
        while columnNumber > 25:
            rem = columnNumber % 26
            columnNumber = columnNumber // 26
            result.append(alphabet[rem])
            columnNumber -= 1

        result.append(alphabet[columnNumber])
        return "".join(result[::-1])
            
        

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder sb = new StringBuilder();
        columnNumber--;
        while (columnNumber > 25) {
            sb.append((char)(columnNumber % 26 + 'A'));
            columnNumber /= 26;
            columnNumber--;
        }
        sb.append((char)(columnNumber + 'A'));
        return sb.reverse().toString();
    }
}